web/backend/src/Server.ts

```diff
@@ -400,14 +400,24 @@ function sendToDela(dataStr: string, req: express.Request, res: express.Response
   // we strip the `/api` part: /api/election/xxx => /election/xxx
   let uri = process.env.DELA_NODE_URL + req.baseUrl.slice(4);
 
-  // in case this is a DKG  init request, we must extract the proxy addr and
-  // update the payload.
-  const regex = /\/evoting\/services\/dkg\/actors$/;
+  // in case this is a DKG setup request, we must update the payload.
+  const regex = /\/evoting\/services\/dkg\/actors/;
```

Comment:
```suggestion
  const regex = /\/evoting\/services\/dkg\/actors$/;
```

Reply:
But that won't match a DKG setup request with the '$' at the end will it (since it's an end of string) and the setup still has the {ElectionID} at the end ie. actors/{ElectionID} ?

Reply:
yes you're right, `/\/evoting\/services\/dkg\/actors\/.*$/` would be the one then. Just to be as much explicit as possible.